src/lib/npmCommand.ts

```diff
@@ -78,28 +79,86 @@ export class NpmCommand {
     return this.npmPkgPath;
   }
 
-  private static npmCli(): string {
+  /**
+   * Return a executable path to this modules reference to npm as
+   * <path to node executable> <path to npm-cli.js>
+   *
+   * @private
+   */
+  private static npmCli(root: string = undefined): string {
+    const nodeBinPath = NpmCommand.findNodeBin(root);
     const pkgPath = NpmCommand.npmPackagePath();
     const pkgJson = fs.readJsonSync(pkgPath) as NpmPackage;
     const prjPath = pkgPath.substring(0, pkgPath.lastIndexOf(path.sep));
-    return path.join(prjPath, pkgJson.bin['npm']);
+    return `"${nodeBinPath}" "${path.join(prjPath, pkgJson.bin['npm'])}"`;
+  }
+
+  /**
+   * Locate node executable and return its full path.
+   * First see if node is on the path, if so, use unqualified name.
+   * If not on the path, try to locate the node version installed with sfdx.
+   * If found, return full path to the executable
+   * If sfdx or node executable cannot be found, an exception is thrown
+   *
+   * @private
+   */
+  private static findNodeBin(root: string = undefined): string {
+    let sfdxPath;
+    if (root) {
+      sfdxPath = root;
+    } else {
+      throw new SfdxError('Plugin root dir is not set', 'PluginRootNotSet');
+    }
```

Comment:
```suggestion
    if (!root) {
      throw new SfdxError('Plugin root dir is not set', 'PluginRootNotSet');
    } 
```
and then just use `root` instead of `sfdxPath` below